config/configmap/inferenceservice.yaml

```diff
@@ -292,7 +292,10 @@ data:
            "memoryLimit": "1Gi",
            "cpuRequest": "100m",
            "cpuLimit": "1",
-           "defaultUrl": "http://default-broker"
+           "defaultUrl": "http://default-broker",
+           "caBundle": "kserve-logger-ca-bundle",
+           "caCertFile": "service-ca.crt",
+           "tlsSkipVerify": false
```

Comment:
I am not sure even though we accept this config, it will work or not because to support global cert from kserve, odh-model-controller overwrites this value with storage-config.